### DISCUSSION TITLE
How do you make two animations for a window in hyprland?

### DISCUSSION THREAD

**Discussion by Kushiko** [hyprwm/Hyprland#8049] [Q&A]
Hello. I want to make a Geeny-style window opening animation. So that the window moves from top to bottom when opened and slowly increases (popin). How can you combine two animations at once? And one more question. How do I make an animation for a single app (rofi)? Thank you in advance)

**Comment by vaxerski** [comment]
you cant

> How do I make an animation for a single app (rofi)?

you read the wiki

**Reply by Kushiko** [reply]
> you cant
> 
> > How do I make an animation for a single app (rofi)?
> 
> you read the wiki

And what about the "double animation"?

**Reply by vaxerski** [reply]
that was what the "you cant" was referring to.

**Reply by Kushiko** [reply]
And for a single app?

**Reply by vaxerski** [reply]
you read the wiki

**Reply by Kushiko** [reply]
I didn't find this on the wiki

**Reply by vaxerski** [reply]
https://wiki.hyprland.org/Configuring/Window-Rules/

**Reply by Kushiko** [reply]
I can't do this for rofi...

**Reply by vaxerski** [reply]
you surely can, I do it for fuzzel.

**Reply by Kushiko** [reply]
Can you give me a line for rofi? `windowrulev2 = animation popin, class:^(rofi)$` Not Working

**Reply by Aqa-Ib** [reply]
Rofi is a layer. Look at the bottom of the page at the Layer Rules section.
